functions/Export-SqlUser.ps1

```diff
@@ -96,7 +96,8 @@ https://dbatools.io/Export-SqlUser
 		[System.Management.Automation.PSCredential]$SqlCredential,
 		[Alias("NoOverwrite")]
 		[switch]$NoClobber,
-		[switch]$Append
+		[switch]$Append,
+		[switch]$IncludeSystemDatabases = $false
```

Comment:
Thank you so much, ben! 

In addition, a switch will auto default to $false so = $false is not needed :) But `[switch]$IncludeSystemDatabases = $false` would require users to do `-IncludeSystemDatabases` so this should change to -NoSystemDatabases if we go that route (i vote we do as well)